freqtrade/optimize/backtesting.py.dev

```diff
@@ -0,0 +1,991 @@
+# pragma pylint: disable=missing-docstring, W0212, too-many-arguments
```

Comment:
I assume this file should either be removed or renamed to `backtesting.py` (if it contains the changes needed for backtesting support of this).

it's a good reminder though that backtesting support for partial sells seems to be completely absent at the moment (this file will not be used by freqtrade).

Reply:
For example without fees,
1   buy 2 @ 10
2   buy 2 @ 5
3  sell 2 @ 6
4   sell 2 @ 9

What is the realized and unrealized profit after order 2 and 3?
What is the realized profit after order 4?

The final profit is 0

Reply:
U0datef